assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -73,6 +76,11 @@ export default function SettingsForm( {
 		select( MODULES_ANALYTICS ).hasFinishedLoadingGTMContainers()
 	);
 
+	const isGA4Connected = useSelect( ( select ) =>
+		select( CORE_MODULES ).isModuleConnected( 'analytics-4' )
+	);
+	const ga4ReportingEnabled = useFeature( 'ga4Reporting' );
+
```

Comment:
IMO, it's appropriate to use the `isGA4DashboardView` here as it does these checks. However, with a couple of changes in the selector:
1. It checks with `isModuleActive`, but we can change it to `isModuleConnected` instead.
2. It checks the dashboard view from the analytics settings object (`ga4Settings.dashboardView === 'google-analytics-4'`). So instead, we can use the `getDashboardView` selector.

Reply:
Hmm. It doesn't seem correct to use `isGA4DashboardView()` here, as we don't need to check the value of `dashboardView` itself, and furthermore the values of `isGA4Connected` and `ga4ReportingEnabled` are checked separately so `isGA4DashboardView()` wouldn't be useful as a replacement for these anyway...